src/services/__service_mocks/api_jokes_random_limitTo_nerdy.ts

```diff
@@ -0,0 +1,17 @@
+import * as nock from 'nock';
+
+const response = {
+  type: 'success',
+  'value': {
```

Comment:
I am used to use linter for such a mistakes. Probably linter rules should be improved.